### PR TITLE
Fix migration guide link

### PR DESCRIPTION
**Pull request by alice-i-cecile**

### CODE REVIEW

File: content/news/2024-02-17-bevy-0.13/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Thanks to **198** contributors, **672** pull requests, community reviewers, and
 For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out our [Quick Start Guide](/learn/book/getting-started/) to try it today. It's free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
 And to see what the engine has to offer hands-on, check out the entries in the [latest Bevy Jam](https://itch.io/jam/bevy-jam-4/entries), including the winner [That's a LOT of beeeeees](https://andrewb330.itch.io/thats-a-lot-of-beeeeees).
 
-To update an existing Bevy App or Plugin to **Bevy 0.13**, check out our [0.12 to 0.13 Migration Guide](/learn/migration-guides/0.12-0.13/).
+To update an existing Bevy App or Plugin to **Bevy 0.13**, check out our [0.12 to 0.13 Migration Guide](/learn/migration-guides/012-013/).
```
Comment (alice-i-cecile) [inline content/news/2024-02-17-bevy-0.13/index.md:18]:
```suggestion
To update an existing Bevy App or Plugin to **Bevy 0.13**, check out our [0.12 to 0.13 Migration Guide](/learn/migration-guides/012-to-013/).
```

File: content/news/2024-02-17-bevy-0.13/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Thanks to **198** contributors, **672** pull requests, community reviewers, and
 For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out our [Quick Start Guide](/learn/book/getting-started/) to try it today. It's free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
 And to see what the engine has to offer hands-on, check out the entries in the [latest Bevy Jam](https://itch.io/jam/bevy-jam-4/entries), including the winner [That's a LOT of beeeeees](https://andrewb330.itch.io/thats-a-lot-of-beeeeees).
 
-To update an existing Bevy App or Plugin to **Bevy 0.13**, check out our [0.12 to 0.13 Migration Guide](/learn/migration-guides/0.12-0.13/).
+To update an existing Bevy App or Plugin to **Bevy 0.13**, check out our [0.12 to 0.13 Migration Guide](/learn/migration-guides/012-to-013/).
```
Comment (alice-i-cecile) [inline content/news/2024-02-17-bevy-0.13/index.md:18]:
```suggestion
To update an existing Bevy App or Plugin to **Bevy 0.13**, check out our [0.12 to 0.13 Migration Guide](/learn/migration-guides/0-12-to-0-13/).
```

**Comment by cart** [comment]
I've been yeeting fixes in via direct commits to avoid process / waiting: https://github.com/bevyengine/bevy-website/commit/6b6d52f81dbf4aee7813204cfcd85bfdab992c40

**Review by TrialDragon — APPROVED** [review]
Yep, that's the proper link now. (I was going to submit a review with it, but I forgot to actually finish review before you fixed it aksdj).

**Pull request closed without merging**